docs/self-managed/operational-guides/backup-restore/backup-and-restore.md

```diff
@@ -9,9 +9,10 @@ keywords: ["backup", "backups"]
 This release introduces breaking changes for [Operate and Tasklist](./operate-tasklist-backup.md), as well as [Optimize](./optimize-backup.md).
 :::
 
-You can use the backup feature of Camunda 8 Self-Managed to regularly back up the state of all of its components (Zeebe, Operate, Tasklist, and Optimize) without any downtime. In case of failures that lead to data loss, you can recover the cluster from a backup.
+You can use the backup feature of Camunda 8 Self-Managed to regularly back up the state of all of its components (Zeebe, Operate, Tasklist, and Optimize) without any downtime (except Web Modeler, see [the Web Modeler backup and restore documentation](./modeler-backup-and-restore.md)).
+In case of failures that lead to data loss, you can recover the cluster from a backup.
 
-A backup of Camunda 8 consists of a backup of Zeebe, Operate, Tasklist, Optimize, and the backup of exported Zeebe records in Elasticsearch. Since the data of these applications are dependent on each other, it is important that the backup is consistent across all components. Therefore, you must take the backup of a Camunda 8 cluster as a whole. The backups of individual components which are taken independently may not form a consistent recovery point. To ensure a consistent backup, follow the process described below.
+A backup of a Camunda 8 cluster consists of a backup of Zeebe, Operate, Tasklist, Optimize, the backup of exported Zeebe records in Elasticsearch. Since the data of these applications are dependent on each other, it is important that the backup is consistent across all components. Therefore, you must take the backup of a Camunda 8 cluster as a whole. The backups of individual components which are taken independently may not form a consistent recovery point. To ensure a consistent backup, follow the process described below.
```

Comment:
The content below only apply to Zeebe, Operate, Tasklist and Optimize but not to Identity, Web Modeler and Keycloak.

@christinaausley Can you think of a good way to clarify this? This whole page is very much centered around the "old" C8 components before Identity and Web Modeler were introduced.

Reply:
So my first question is should this only be in versioned docs, then? If this is very much relevant to content today (I assume it is, but double checking), then I would just phrase this exactly as you explained it -- perhaps add an admonition that says something like "This content is related to X components and their functionality prior to the launch of Identity and Web Modeler."

But the caveat is that this leaves me with the question "If this is centered around "old" C8 components, is this information outdated? Should I be using something that is more modern, or is this content just unrelated to certain components?

Regardless, as far as phrasing goes:

A backup of a Camunda 8 cluster consists of a backup of Zeebe, Operate, Tasklist, Optimize, and exported Zeebe records in Elasticsearch. Since the data of these applications are dependent on each other, it is important that the backup is consistent across all components. The backups of individual components taken independently may not form a consistent recovery point. Therefore, you must take the backup of a Camunda 8 cluster as a whole. To ensure a consistent backup, follow the process described below.

Reply:
> If this is very much relevant to content today (I assume it is, but double checking)

Yes, it is relevant to content today **and** the versioned docs. The problem with this page is that it has never really been touched since it was initially written so it is focused on "cluster components" (i.e. Zeebe, Tasklist, Operate, Optimize). Web Modeler is a special case here as it is cluster-independent which is why I'd refrain from adding an admonition stating that the content is outdated - we should aim to describe the current status quo.

Additionally, there is still a note admonition at the top of the page stating that "this release introduces breaking changes for Operate and Tasklist, as well as Optimize" which as far as I know is **not** the case in the current version. The note has been added in February 2023 (see git blame).

Reply:
@christinaausley @wollefitz I will add a separate issue to the backlog to revisit this page, top to bottom. Thank you for raising this!